src/server/rpc/procedures/chart/chart.js

```diff
@@ -0,0 +1,105 @@
+const ApiConsumer = require('../utils/api-consumer'),
+    ChartNode = require('chartjs-node');
+
+let chart = new ApiConsumer('chart');
+let chartNode = new ChartNode(600, 600);
+let defaultColor = 'rgba(74, 108, 212, 0.8)';
+
+let defaultOption = (xAxis, yAxis, title) => {
+    return {
+        scales: {
+            xAxes: [{
```

Comment:
That's in the Chart.js documentation
![image](https://user-images.githubusercontent.com/18899970/27657389-2204e11e-5c12-11e7-97e9-4fb9eff8ced4.png)